app/src/main/java/com/example/dine_and_donate/ProfileActivity.java

```diff
@@ -131,4 +138,31 @@ public boolean onCreateOptionsMenu(Menu menu) {
         getMenuInflater().inflate(R.menu.activity_main, menu);
         return true;
     }
+
+    private void createUserModel () {
+        final FirebaseUser user = mAuth.getCurrentUser();
+
+        FirebaseDatabase mDatabase = FirebaseDatabase.getInstance();
+        DatabaseReference mRef = mDatabase.getReference();
+
+        DatabaseReference ref = mRef.child("users"); // reference to users in database
```

Comment:
You don't really need mRef here, you could just do 

DatabaseReference ref = mDatabase.getReference().child("users");